### ISSUE TITLE
Problems running on Mac OS Ventura

### ISSUE THREAD

**Issue by Phl12** [AdguardTeam/AdGuardHome#6063]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Darwin (aka macOS), AMD64 (aka x86_64)

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

0.107.35

### Action

Hello,

I was running AdGuard Home on a mac mini (2014) with Mac OS Monterey and all was working perfectly.
I have just changed my mac mini and now using a mac mini 2018 (still using an Intel processor) running on Mac OS Ventura, on which I re-installed (through automatic install) AdGuard Home - the same way I installed it previously.
Since then, the devices (iPhone and iPad) using AdGuard Home as DNS behave badly with network. Some "normal" (non blocked) web sites can't be accessed or first http request is very very long. Some apps that were running all fine when I was using previous configuration are now blocked when accessing Internet.
Tried to reinstall AdGuard Home but didn't solve the problem.

What are the info / logs you would need to investigate my problem?

### Expected result

Normal filtering 

### Actual result

Network hang, unable to reach some normal sites.

**Comment by ainar-g** [comment]
The [AGH verbose logs](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) as well as the results of `dig` queries (for example, `dig IN A 'www.example.com' @<AGH_IP_ADDRESS>`), assuming you can perform the latter from one of the affected devices.  As well as how you've configured your devices to begin with.  (Could it be that AGH's IP address has changed and the devices cannot resolve the domain name now?)

**Comment by TNOTA** [comment]
Running on Ventura on a Mac mini (intel) for over a year now, no issues at all.  Runs perfectly.

**Comment by Phl12** [comment]
Hello,

I changed the log mode to be able to provide you with verbose logs but I
was not able to reproduce the problem.
I changed the log mode back and it continues to run fine.
I have changed other parameters on MacOS related to energy savings which
may better guarantee that the server never sleeps (this setup location has
spread in different parameters under Ventura) and this might have been the
cause of my problem (so not a problem with AGH). I propose you close the
ticket and I will reopen it if problem comes back.
Thank you for your support.

Philippe Legrand


Le mar. 1 août 2023 à 12:47, Ainar Garipov ***@***.***> a
écrit :
